metapool/tests/test_plate.py

```diff
@@ -563,6 +565,110 @@ def test_sample_plate_has_no_spaces(self):
         self.assertEqual(context, expected)
 
 
+class PlateReplicationTests(TestCase):
+    def setUp(self):
+        self.input_df = pd.read_csv('metapool/tests/data/input_plate.tsv',
+                                    sep='\t', dtype=str)
+
+    def _frames_equal(self, obs, exp):
```

Comment:
Well, I do _not_ think this is necessary as if the frames are equal, they are simply equal and there is nothing to test against None but if they are not the error is raised. Another way to say this, nowhere (that I can tell) the code below actually testes for `self.assertFalse(_frames_equal....`. Can you simply use `assert_frame_equal`?

Reply:
I suppose you're right. I confirmed that AssertionError raised by assert_frame_equal is a standard Error and that the standard self.assert*() methods also return the same type.